### ISSUE TITLE
Need adapter to expose 'joinedTime'

### ISSUE THREAD

**Issue by kylebakerio** [networked-aframe/networked-aframe#219]
I need an equivalent of `NAF.connection.joinedTime`. In NAF, the server join time is used by clients to determine whether they should initiate a connection or not (whoever connected first starts the handshake).

My client side code also needs this functionality, but as far as I can tell, this join time is not exposed by NAF. I can replicate the functionality, but that's kind of silly. I can get the join time of other clients, but not my own, and if I try to save my own and then compare, it's always a little later--later enough, that both clients think they were second if they both join at the same time. Which means that I'm left with having both users send them each others' join times... Which feels absurd.

If a new release is coming, it's a one line fix to just throw that onto the publicly accessible NAF.connection object. I had done it in my fork, but since there is a rumor that an update is coming soon, I figured I'd throw this request in here. (Also glad to do a pull request, but I just don't see any insight into how to contribute here.)

**Comment by vincentfretin** [comment]
With the easyrtc adapter, you have
`NAF.connection.adapter._myRoomJoinTime`
and also
`NAF.connection.adapter._getRoomJoinTime(clientId)`

`NAF.connection.adapter._getRoomJoinTime(NAF.clientId)`
is the same as 
`NAF.connection.adapter.easyrtc.getRoomOccupantsAsMap(NAF.room)[NAF.clientId].roomJoinTime`

This is specific to easyrtc so probably why it's private and not part of the public adapter interface.

**Comment by vincentfretin** [comment]
Other adapters like janus doesn't have this information.

**Comment by kylebakerio** [comment]
At the time I was using the webrtc adapter I think. I also later figured out ways around it, though they don't come to mind right now. Still, it would have been nice to have, and would be good to document.

**Comment by vincentfretin** [comment]
PR welcome to write some doc about it :) (after I resurrect the easyrtc adapter)

**Comment by kylebakerio** [comment]
yeah, I got paid to give private lessons in part just to explain to someone else what adapters are and how adapters work in NAF. Probably should add some clarity there. will definitely do.
